src/host/VtIo.cpp

```diff
@@ -247,25 +234,6 @@ void VtIo::CreatePseudoWindow()
     }
 }
 
-void VtIo::SetWindowVisibility(bool showOrHide) noexcept

```

Comment:
📝 Who converts pseudo-window visibility changes into output-stream VT now?

Reply:
`InteractivityFactory::_WritePseudoWindowCallback`. Ideally it should be a `WriteDTTERM` member function or something though.